### ISSUE TITLE
Use routeable tiles

### ISSUE THREAD

**Issue by xivk** [openknowledgebe/equalstreetnames#12]
We have built this last year:

https://github.com/openplannerteam/routable-tiles

It publishes the entire OSM road network as JSON-LD, once we have this updating continuously it could be used to make this map:

- update live
- globally available

**Comment by jbelien** [comment]
Ah Oh ! I guess it would indeed improve the performance tremendously !

But I don't really see on how to include this in the current process.
Are those tiles hosted somewhere already ?
How can I add information (gender/wikidata) on the road network from those tiles ?

**Comment by xivk** [comment]
This is an example:

https://tiles.openplanner.team/planet/14/8449/5558/

They are not there now but we could include the wikidata tags (it would make a lot of sense to do that) and then you could lookup the data from wikidata and show the streets in the correct way.

**Comment by xivk** [comment]
Actually I think the wikidata tags would be there, except that the data hosted now is out of data (somewhere beginning of 2019).

**Comment by xivk** [comment]
This tile for example has the wikidata tags:

https://tiles.openplanner.team/planet/14/8392/5496/

**Comment by jbelien** [comment]
That's most definitely a good idea but will require quite some work :P

**Comment by xivk** [comment]
Yes yes, not saying this should be the focus, it could be a long/mid term thing if ever.
